.github/workflows/build.yml

```diff
@@ -51,3 +51,9 @@ jobs:
       - name: Build and unit test
         if: ${{ ! contains( github.event.pull_request.labels.*.name, 'NoTestNeeded') }}
         run: mvn clean install --no-transfer-progress -Pspark-${{ matrix.spark }} -Dspan.scale.factor=10
+      - name: Build and unit test Atum Examples Scala ${{ matrix.scala }} and Spark ${{ matrix.spark }}
+        working-directory: ./examples/atum-examples
+        if: ${{ ! contains( github.event.pull_request.labels.*.name, 'NoTestNeeded') }}
+        run: |
+          mvn scala-cross-build:change-version -Pscala-${{ matrix.scala }}
+          mvn clean install --no-transfer-progress -Pspark-${{ matrix.spark }} -Dspan.scale.factor=10
```

Comment:
Why not create a second action? `build_examples.yml`

Reply:
I think this has multiple advantages:
 - the setup matrix is already there (the examples are thus built for Scala 2.11 + Spark2.4, Scala 2.12 + Spark 2.4, and Scala 2.12 + Spark 3.1.
 - the core Atum is already built and installed, a second workflow would have to do it all over.

So a step made more sense to me, but I am by no means an actions expert, so feel free to prove me wrong.

Reply:
I forgot you need cached core